### PR TITLE
display reported hashrate when available, average hashrate when not a…

### PR DESCRIPTION
**Pull request by hendoban**
…vailable

### CODE REVIEW

File: public/locales/en-US/dashboard.json [hunk 1]
```diff
@@ -193,8 +193,8 @@
       "table_head": {
         "name": "Name",
         "reported_hashrate": "Reported Hashrate",
-        "current_e_hashrate": "Current E. Hashrate",
-        "average_e_hashrate": "Average Effective Hashrate",
+        "current_e_hashrate": "Current Hashrate",
+        "average_e_hashrate": "Average Hashrate",
         "valid": "Valid Shares",
```
Comment (AlexSSD7) [inline public/locales/en-US/dashboard.json:198]: Better to revert this because it will be unclear if something is about effective or reported hashrate.

**Review by AlexSSD7 — CHANGES_REQUESTED** [review]

File: public/locales/en-US/dashboard.json (hunk 1 above)
Comment (hendoban) [reply public/locales/en-US/dashboard.json]: I can only fit the E. in due to space constraints but they have been added back in.

**Review by AlexSSD7 — COMMENTED** [review]
Effectiveness % was removed?

**Comment by hendoban** [comment]
percentage is put into a tooltip when reported is present due to space constraints.

image

when reported is not present I can put a tooltip with a percentage of average vs current if you'd like or put it on current but when reported isn't there its a nicehash renter.

**Review by AlexSSD7 — APPROVED** [review]
LGTM

**Pull request merged**